### ISSUE TITLE
Sorting of Connected Components for Directed Graphs

### ISSUE THREAD

**Issue by 0ddoes** [networkx/networkx#5564]
It is not possible to sort the connected_graphs() of a Directed Network as it raises the ```nx.NetworkXNotImplemented``` Error for me. I wanted to plot only the most connected subgraph , however it doesn't allow to sort. Is this normally not implemented for Directed Graphs?

**Comment by rossbar** [comment]
I'm not quite sure I understand the question. If you mean you want a list of the strongly connected components of a directed graph in order of component size you could do something like:

```python
>>> scc = sorted(nx.strong_connected_components(G), key=len)
```

I'm not sure this is what you had in mind though... if not, it would help to have a minimal example of the behavior you're looking for!

**Comment by 0ddoes** [comment]
Thank you for the response. I don't remember why exactly the error occured, but I did solve it using ```max(nx.strongly_connected_components(G), key=len)```. At the time I got the error when using sorted on Directed Graphs, and was not able to get the subgraph from the parent Graph. 

I included it in #72 in nx_guides
